### ISSUE TITLE
Importing in ES6 is broken

### ISSUE THREAD

**Issue by ekryski** [feathersjs/feathers-client#14]
```js
import feathers from 'feathers-client';

const app = feathers().configure(feathers.primus({
  transformer: 'sockjs',
  pathname: '/primus',
}));

const locationService = app.service('location');
```

Errors with `Uncaught TypeError: _feathersClient2.default.primus is not a function`.

This is most likely related to how we were previously compiling the ES6 modules without the babel module exports shim.

**Comment by daffl** [comment]
Do you have latest v0.3.3? I think that's what @marshallswain fixed in #13

**Comment by daffl** [comment]
I just double checked that this is fixed in 0.3.3.
